.github/workflows/cldk_workflow.yaml

```diff
@@ -0,0 +1,60 @@
+name: CI for CLDK Notebooks
+
+on:
+  push:
+    branches:
+      - main
+#    paths:
+#      - '.github/notebook_lists/cldk_notebooks.txt'
+  pull_request:
+    branches:
+      - main
+#    paths:
+#      - '.github/notebook_lists/cldk_notebooks.txt'
```

Comment:
these lines will specify to only run the test if one of the files listed here have been changed, i will update with the paths of the notebooks in the CLDK directory